Orange/base.py

```diff
@@ -88,9 +88,15 @@ class Learner(ReprableWithPreprocessors):
     #: A sequence of data preprocessors to apply on data prior to
     #: fitting the model
     preprocessors = ()
-    FittedParameter = namedtuple(
-        "FittedParameter",
-        ["parameter_name", "label", "tick_label", "type", "min", "max"]
+    FittedParameter = NamedTuple(
+        "FittedParameter", [
+            ("name", str),
+            ("label", str),
+            ("tick_label", str),
+            ("type", Type),
+            ("min", Union[int, NoneType]),
+            ("max", Union[int, NoneType]),
+        ]
```

Comment:
What I had in mind was

```python
class FittedParameter(NamedTuple):
    name: str
    label: str
    tick_label: str = ""
    type_: type = int
    min: Optional[int] = None
    max: Optional[int] = None
```

No need to change it; I find this easier on the eye, but it's a matter of taste.